src/EA.Weee.RequestHandlers/AatfEvidence/SetNoteStatusRequestHandler.cs

```diff
@@ -53,6 +53,18 @@ public async Task<Guid> HandleAsync(SetNoteStatusRequest message)
                     organisationAddress.CountyOrRegion,
                     organisationAddress.Postcode,
                     null));
+
+                if (evidenceNote.NoteType == Domain.Evidence.NoteType.TransferNote)
```

Comment:
@GuidoWorkDev is there a unit test to cover the case where the note is an evidence note... the approved transferred address should be null?